x-pack/solutions/search/plugins/enterprise_search/server/deprecations/routes.ts

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Should this file be in `server/routes/deprecations.ts` instead? possibly `server/routes/enterprise_search/deprecations.ts` 

I only mention it because thats the established pattern of where we put all other routes.

Reply:
if you'd prefer, sure! I wanted to keep all the deprecations stuff co-located, since it won't ever move up into 9.x, so I didn't want to add it to existing routes files. But a new route file next to the others would be fine, if that's the better pattern.